Simple-Projects/Simple_Projects/Basic/Complex/main.cpp

```diff
@@ -1,9 +1,9 @@
-#include <stdio.h>
+#include <iostream>
```

Comment:
Make sure to include C headers in the form <c*>, not <*.h>, because they differ in C++ from C
And stdio is not used here